lib/transition/import/whitehall_orgs.rb

```diff
@@ -8,13 +8,6 @@ module Import
     class WhitehallOrgs
       include Enumerable
 
-      # org_yaml_path is only for test usage, so that we can pass in a fixture
-      # rather than calling a live API. In production this always calls the
-      # live API.
-      def initialize(org_yaml_path = nil)
-        @org_yaml_path = org_yaml_path
-      end
-
       def organisations
```

Comment:
I wonder if `all` might have been a better method name here (again predates this PR sorry)